src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-mon calendar--month-length-31">
```

Comment:
Avoid using multiple modifiers in BEM that dictate the state of the component (`calendar--start-day-mon` and `calendar--month-length-31`). It's better to use JavaScript to add these classes dynamically based on the state or consider a different approach to manage these states.